### PR TITLE
fix: pfe-tabs issue with active border color on horizontal/wind (#1585) 

### PR DESCRIPTION
**Pull request by bonjo7**
## Component updates: pfe-tabs

On horizontal/wind tabs, the active border is not maintaining its color (#06c) when clicking between tabs. The active state is being being overwritten by the focus state border.

### Related issues

- #1585

### Preview

Link(s) to demo page(s) where this element can be viewed:
- [Demo page](https://deploy-preview-1689--patternfly-elements.netlify.app/components/tabs/)

### What has changed and why

- Removed `host(:focus),` css element from horizontal/wind tabs

### Testing instructions

1. Go to https://patternflyelements.com/components/tabs/
2. Click on one of the tab elements in the horizontal/wind tabset
3. Correct color on active tab element bottom border

#### Browser requirements

Your component should work in all of the following environments:

- [x] Latest 2 versions of Edge
- [x] Internet Explorer 11 (should be useable, not pixel perfect)
- [x] Latest 2 versions of Firefox (one on Mac OS, one of Windows OS)
- [x] Firefox 78 (or latest version for Red Hat Enterprise Linux distribution)
- [x] Latest 2 versions of Chrome (one on Mac OS, one of Windows OS)
- [x] Latest 2 versions of Safari
- [x] Android mobile device (such as the Galaxy S9)
- [x] Apple mobile device (such as the iPhone X)
- [x] Apple tablet device (such as the iPhone Pro)

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [x] Browser testing passed.
- [x] Changelog updated.
- [ ] Approved by designer.

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

**Comment by castastrophe** [comment]
@bonjo7 Thanks for tackling this issue!  I've pasted the template into your description, please go through and fill that out when you get a chance.  We also need a changelog update from you when you're able, something like this at the top:

```
# 1.9.4 (TBD)

- [](https://github.com/patternfly/patternfly-elements/commit/) fix: pfe-tabs issue with active border color on horizontal/wind (#1585)
```

**Comment by castastrophe** [comment]
I have reviewed the code update and branch tested ✅

**Comment by bonjo7** [comment]
Hey @castastrophe thanks for the feedback, I have reviewed the above, one question should the change log just go at the top of the description?

**Comment by castastrophe** [comment]
It should go in the CHANGELOG-1.x.md file at the top of the file. Thank you so much!

**Comment by bonjo7** [comment]
Thank you @castastrophe , I have added to change log and noted for any future PR's from me

File: CHANGELOG-1.x.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
+# 1.9.4 (2021-07-08)
```
Comment (castastrophe) [inline CHANGELOG-1.x.md:1]:
```suggestion
# 1.10.1 (2021)
```

**Review by castastrophe — APPROVED** [review]

**Pull request merged**